include/nbl/builtin/glsl/workgroup/shared_blur.glsl

```diff
@@ -0,0 +1,16 @@
+#ifndef _NBL_BUILTIN_GLSL_WORKGROUP_SHARED_BLUR_INCLUDED_
+#define _NBL_BUILTIN_GLSL_WORKGROUP_SHARED_BLUR_INCLUDED_
+
```

Comment:
this include is in the wrong place and "namespace", should be in `ext/CentralLimitBoxBlur`